backend/src/graphql/model/User.ts

```diff
@@ -3,19 +3,24 @@ import { KlickTipp } from './KlickTipp'
 import { User as dbUser } from '@entity/User'
 import Decimal from 'decimal.js-light'
 import { FULL_CREATION_AVAILABLE } from '../resolver/const/const'
+import { UserContact } from './UserContact'
 
 @ObjectType()
 export class User {
   constructor(user: dbUser, creation: Decimal[] = FULL_CREATION_AVAILABLE) {
     this.id = user.id
     this.gradidoID = user.gradidoID
     this.alias = user.alias
-    this.email = user.email
+    this.emailId = user.emailId
+    if (user.emailContact) {
+      this.email = user.emailContact.email
```

Comment:
currently yes, but you are right we should hide it in the future.
I kept it to reduce more change efforts especially in the frontend, because the email is still presented/used there.